internal/controller/bsl_test.go

```diff
@@ -1518,6 +1518,8 @@ func TestDPAReconciler_ValidateBackupStorageLocations(t *testing.T) {
 }
 
 func newContextForTest(name string) context.Context {
+	// todo: used a lot, combing needed
```

Comment:
combing = combining?

Reply:
Oh, I meant combing through the places where this func is called
all the func calls pass the name param, so was not sure if I should remove this param from all those calls. 

there's one other commit like this where I set the unused param to the blank identifier, would like inputs on if it's better to remove it altogether

Reply:
based on the return type of context.Context where I don't think it's possible to add names here, I think this one can just remove the unused param.